server/index.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+const swaggerUi = require('swagger-ui-express');
+const mongoose = require('mongoose');
+const specs = require('../swagger');
+
+const app = express();
+/**
+ * @swagger
+ * /:
+ *   get:
+ *     summary: Welcome to Banka API
+ *     description: Returns a welcome message for Banka API.
+ *     responses:
+ *       200:
+ *         description: A successful response with the welcome message.
+ *         content:
+ *           application/json:
+ *             schema:
+ *               type: object
+ *               properties:
+ *                 message:
+ *                   type: string
+ *                   example: Welcome to Banka API
+ */
```

Comment:
Can we extract this into a file of its own? This would get messy when we get more routes in. See Send-IT or my most recent repo bare-foot